language/en/contributions.php

```diff
@@ -153,7 +153,8 @@
 	'NEW_CONTRIBUTION'						=> 'New Contribution',
 	'NEW_REVISION'							=> 'New Revision',
 	'NEW_REVISION_READY'					=> 'New revision ready for processing.',
-	'NEW_REVISION_REPACK_COMPLETE'			=> 'New revision has been repacked to ensure packaging compliance. Click <em>Continue</em> to process through EPV.',
+	'NEW_REVISION_REPACK_COMPLETE'			=> 'New revision has been repacked to ensure packaging compliance. Click <em>Continue</em> to process through EPV. ',
+	'NEW_REVISION_REPACK_OVERSIZE'			=> 'If you are unable to submit your contribution, please contact a member of the <a href="https://www.phpbb.com/community/memberlist.php?mode=group&g=228811">Customisations Team</a>.',
```

Comment:
Let's not make this specific to our site. Maybe change to this:

> If you are unable to submit your contribution due to a system error, please contact us through your contribution's <a href="%s">validation discussion topic</a> for further assistance.

And add a link to the contribution's validation discussion topic. That's more open-ended for the user and does not make them decide who would be the one to speak to. Would need to verify that the validation discussion topic is actually created at contribution creation time as it might be getting created when creating a revision.

Reply:
Changed at https://github.com/phpbb/customisation-db/pull/291/commits/9cd178635cd6131b244878362717d05d75ec5990

Tested on a new contribution too, and it creates the topic before the revision.